libraries/iio/iio.c

```diff
@@ -102,6 +102,7 @@ struct element_info {
  * with a "iio_device *iio" that describes capabilities of the device.
  */
 struct iio_interface {
+	const char		dev_id[10];
```

Comment:
Is this long enough ?

Reply:
Added comment to explain this size. Should be enough for 999 devices :)